mitodl/ocw-studio
# Add website collection editing functionality

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #357 

#### What's this PR do?

this adds functionality for editing basic attributes of WebsiteCollection records (i.e. their title, description) and for therefore creating new ones.

This does not yet include editing entries in a collection, that will be covered for #376

#### How should this be manually tested?

Visit the collections list page (`/collections`). If you click 'Add' you should get a drawer where you can create a new collection. Then in the list page if you click on the title for a collection you should open an editing drawer where you can change it's title and whatnot.

#### Screenshots (if appropriate)

<img width="1014" alt="Screen Shot 2021-07-13 at 10 48 06 AM" src="https://user-images.githubusercontent.com/6207644/125473339-eb6307af-e4da-484f-af9b-0e3241bd9720.png">
<img width="594" alt="Screen Shot 2021-07-13 at 10 48 13 AM" src="https://user-images.githubusercontent.com/6207644/125473340-2858f365-ac3c-4f0c-8333-ad1c26ba6e89.png">
<img width="555" alt="Screen Shot 2021-07-13 at 10 48 21 AM" src="https://user-images.githubusercontent.com/6207644/125473343-cbe81632-512e-40cd-ae91-b84c040a951b.png">
